File: Atividade02/atv02PHP/qstMain.php
<?php
$n1 = intval(readline("Coloque o primeiro número: "));
$n2 = intval(readline("Coloque o segundo número: "));
$op = readline("Coloque a operação: ");

$cal = new Calculo();
$cal->Soma($n1, $n2);
$resultado = $cal->Operacao($op, $n1, $n2);
echo "O resultado é: $resultado\n";
$hora = $cal->Hora();
echo "$hora\n";

File: Atividade02/atv02Python/qstMain.py
from Calc import Calculo

n1 = int(input("Coloque o primeiro número: "))
n2 = int(input("Coloque o segundo número: "))
op = input("Coloque a operação: ")

cal = Calculo()
cal.Soma(n1,n2)
cal.Operacao(op, n1, n2)
hora = cal.Hora()
print(hora)